spec/script/script_spec.rb

```diff
@@ -394,6 +419,18 @@ module Cosmos
       end
     end
 
+    describe "play_wav_file" do
+      it "should play a wav file if Qt is available" do
+        module Qt
```

Comment:
This looks like the culprit.  Defining Qt breaks part of top_level.rb.   I will fix.